### PR TITLE
Dashboards Assistant update sidebar content

### LINKED ISSUE #6706: [DOC]
**What do you want to do?**

- [x] Request a change to existing documentation
- [x] Add new documentation
- [ ] Report a technical problem with the documentation
- [ ] Other 

**Tell us about your request.** Provide a summary of the request and all versions that are affected.

 The container of dashboard assistant will be updated from flyout to sidecar.  Previously, the assistant would only be on the right side of the dashboard by default, which did not work well with some visalization flyouts. Now the assistant integrates the sidecar service, It supports three dock modes, left, right and takerover. It is isolated from the dashboard and supports resizable.

I'm not sure if we need to add some documents or there are some content or screenshots in existing documents need to be updated, so I created this issue. please let me know if it's required.

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5920
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5620
https://github.com/opensearch-project/dashboards-assistant/pull/164

### PR DESCRIPTION
**Pull request by vagimeli**
### Description
Adds verbiage about left sidebar

### Issues Resolved
Closes #6706 

### Checklist
- [X] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by vagimeli** [comment]
@raintygao Please review to confirm content changes are accurate. Thanks!

File: _dashboards/dashboards-assistant/index.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ For information about configuring OpenSearch Assistant through the REST API, see
 
 ## Using OpenSearch Assistant in OpenSearch Dashboards
 
-The following tutorials guide you through using OpenSearch Assistant in OpenSearch Dashboards. OpenSearch Assistant can be viewed full frame or in the right sidebar. The default is sidebar. To view full frame, select the frame icon {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/frame-icon.png" class="inline-icon" alt="frame icon"/>{:/} in the toolbar. 
+The following tutorials guide you through using OpenSearch Assistant in OpenSearch Dashboards. OpenSearch Assistant can be viewed full frame or in the sidebar. The default view is in the right sidebar. To view in the left sidebar or full frame, select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/frame-icon.png" class="inline-icon" alt="frame icon"/>{:/} icon in the toolbar.
```
Comment (natebower) [inline _dashboards/dashboards-assistant/index.md:63]:
```suggestion
The following tutorials guide you through using OpenSearch Assistant in OpenSearch Dashboards. OpenSearch Assistant can be viewed in full frame or in the sidebar. The default view is in the right sidebar. To view the assistant in the left sidebar or in full frame, select the {::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/frame-icon.png" class="inline-icon" alt="frame icon"/>{:/} icon in the toolbar. 
```

**Review by natebower — COMMENTED** [review]
@vagimeli Just a few minor changes. Thanks!

**Review by natebower — APPROVED** [review]
Approved with noted changes.

**Pull request merged**